landlab/components/flow_accum/flow_accumulator.py

```diff
@@ -473,7 +476,7 @@ class DepressionFinderAndRouter to the parameter `depression_finder`.
     >>> z += 0.01 * mg.node_y
     >>> mg.at_node['topographic__elevation'].reshape(mg.shape)[2:5, 2:5] *= 0.1
     >>> fa = FlowAccumulator(mg, 'topographic__elevation',
-    ...                        flow_director=FlowDirectorSteepest,
+    ...                        flow_director='FlowDirectorD8',
```

Comment:
Should we keep default as D4?

Reply:
This is just a docstring example. Since I'm not specifying the routing method for DepressionFinderAndRouter here I need to make the FlowDirector be D8 otherwise the example will throw an error that says these two methods are incompatible.